miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultSampleService.java

```diff
@@ -226,6 +233,10 @@ public Long create(Sample sample) throws IOException {
           throw new IOException(e.getMessage(), e);
         }
         validateHierarchy(detailed);
+
+        if (detailed.getDetailedQcStatus() != null) {
```

Comment:
This field is present on Identities too, so should be treated the same (outside of the `if (!isIdentitySample...` block). Alternately, it could be done in `loadChildEntities` instead, which is called during both create and update